src/plugins/syscalls/win.cpp

```diff
@@ -225,6 +225,15 @@ static event_response_t syscall_cb(drakvuf_t drakvuf, drakvuf_trap_info_t* info)
 
     wr->tid = info->proc_data.tid;
     wr->w = w;
+    if ( 4 == s->reg_size )
+    {
+        wr->stack_fingerprint = info->regs->rsp + 4 /*return address*/ + 4 * nargs;
```

Comment:
Please change the inline comment to be in the previous line

Reply:
Sorry i don't know what comment to add.  Like this?
```c
   if ( 4 == s->reg_size )
    {
        // Save the value of the stack pointer when the function returns
        wr->stack_fingerprint = info->regs->rsp + 4 + 4 * nargs;
    }
    else
    {
        // Save the value of the stack pointer when the function returns
        wr->stack_fingerprint = info->regs->rsp + 8;
    }
```
Can you give me some suggestions? And I don't know if this variable name **stack_fingerprint** is suitable.